.github/workflows/palette.yml

```
@@ -0,0 +1,30 @@
+name: Palette
+
+on:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # Run every day at midnight UTC
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

really tiny nitpick but this'll give us node 16 deprecation warnings iirc
```suggestion
      - uses: actions/checkout@v4
```